torch/_dynamo/variables/torch_function.py

```diff
@@ -682,8 +696,15 @@ def var_getattr(self, tx: "InstructionTranslator", name):
                     return getter_var.call_function(tx, [], {})
 
                 elif attr_is_overriden:
-                    unimplemented(
-                        f"Currently only support accessing overridden attributes that are functions or properties, but got {type(attr)}"  # noqa: B950
+                    unimplemented_v2(
+                        gb_type="Tensor subclass attribute access",
```

Comment:
```suggestion
                        gb_type="Tensor subclass non-function/proprety overriden attribute access",
```

Reply:
Added "Unsupported" in front, since the supported set here might change frequently.